tests/suites/test_suite_psa_crypto_pake.function

```diff
@@ -1041,3 +1059,119 @@ exit:
     PSA_DONE();
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:PSA_WANT_ALG_JPAKE:PSA_ALG_SHA_256 */
+void pake_input_getters_user()
+{
+    psa_pake_cipher_suite_t cipher_suite = psa_pake_cipher_suite_init();
+    psa_pake_operation_t operation = psa_pake_operation_init();
+    const uint8_t user[] = "server";
+    const size_t user_len = strlen("server");
```

Comment:
`ARRAY_LENGTH` won't work in this case as it will include `null` char for string termination. This is why `strlen` is used.
I will use the following version if `strlen` is a problem:
```
    const uint8_t peer[] = { 's', 'e', 'r', 'v', 'e', 'r' };
    const size_t peer_len = sizeof(peer);
```